manager/src/main/java/org/openremote/manager/system/HealthService.java

```diff
@@ -119,63 +108,63 @@ public void init(Container container) throws Exception {
-                        ROUTE_ID_TAG, route.getId());
-                }
-            });
-            micrometerRoutePolicyFactory.setMeterRegistry(container.getMeterRegistry());
-            brokerService.getContext().addRoutePolicyFactory(micrometerRoutePolicyFactory);
+//            // Configure camel metrics collection here as need access to the custom registry
+//            MicrometerRoutePolicyFactory micrometerRoutePolicyFactory = new MicrometerRoutePolicyFactory() {
+//                @Override
+//                public RoutePolicy createRoutePolicy(CamelContext camelContext, String routeId, NamedNode routeDefinition) {
+//                    SedaMicrometerRoutePolicy answer = new SedaMicrometerRoutePolicy();
+//                    answer.setMeterRegistry(getMeterRegistry());
+//                    answer.setPrettyPrint(isPrettyPrint());
+//                    answer.setDurationUnit(getDurationUnit());
+//                    answer.setNamingStrategy(getNamingStrategy());
+//                    return answer;
+//                }
+//            };
+//            micrometerRoutePolicyFactory.setNamingStrategy(new MicrometerRoutePolicyNamingStrategy() {
+//                @Override
+//                public String getName(Route route) {
+//                    return OR_CAMEL_ROUTE_METRIC_PREFIX;
+//                }
+//
+//                @Override
+//                public String getExchangesSucceededName(Route route) {
+//                    return OR_CAMEL_ROUTE_METRIC_PREFIX + "_succeeded";
+//                }
+//
+//                @Override
+//                public String getExchangesFailedName(Route route) {
+//                    return OR_CAMEL_ROUTE_METRIC_PREFIX + "_failed";
+//                }
+//
+//                @Override
+//                public String getExchangesTotalName(Route route) {
+//                    return OR_CAMEL_ROUTE_METRIC_PREFIX + "_total";
+//                }
+//
+//                @Override
+//                public String getFailuresHandledName(Route route) {
+//                    return OR_CAMEL_ROUTE_METRIC_PREFIX + "_failed_handled";
+//                }
+//
+//                @Override
+//                public String getExternalRedeliveriesName(Route route) {
+//                    return OR_CAMEL_ROUTE_METRIC_PREFIX + "_redeliveries";
+//                }
+//
+//                @Override
+//                public Tags getTags(Route route) {
+//                    return Tags.of(
+//                        ROUTE_ID_TAG, route.getId());
+//                }
+//
+//                @Override
+//                public Tags getExchangeStatusTags(Route route) {
+//                    return Tags.of(
+//                        ROUTE_ID_TAG, route.getId());
+//                }
+//            });
+//            micrometerRoutePolicyFactory.setMeterRegistry(container.getMeterRegistry());
+//            brokerService.getContext().addRoutePolicyFactory(micrometerRoutePolicyFactory);
```

Comment:
Perhaps we should all the commented code should also have been from the `HealthService`?